demos/inpage/src/EthereumApi.tsx

```diff
@@ -315,9 +316,16 @@ export default class EthereumApi {
         initCode = '0x';
       }
 
-      const feeData = await this.#waxInPage.ethersProvider.getFeeData();
-      assert(feeData.maxFeePerGas !== null);
-      assert(feeData.maxPriorityFeePerGas !== null);
+      let { maxFeePerGas, maxPriorityFeePerGas } =
+        await this.#waxInPage.ethersProvider.getFeeData();
+
+      assert(maxFeePerGas !== null);
+      assert(maxPriorityFeePerGas !== null);
```

Comment:
Any chance these could be `undefined`, or have a value of `0` and thus be invalid? If so: 
```suggestion
      assert(!!maxFeePerGas);
      assert(!!maxPriorityFeePerGas);
```